clients/src/main/java/org/apache/kafka/clients/admin/ScramMechanism.java

```diff
@@ -49,8 +53,11 @@ public static ScramMechanism fromType(byte type) {
      *     Salted Challenge Response Authentication Mechanism (SCRAM) SASL and GSS-API Mechanisms, Section 4</a>
      */
     public static ScramMechanism fromMechanismName(String mechanismName) {
-        ScramMechanism retvalFoundMechanism = ScramMechanism.valueOf(mechanismName.replace('-', '_'));
-        return retvalFoundMechanism != null ? retvalFoundMechanism : UNKNOWN;
+        String normalizedMechanism = mechanismName.replace('-', '_');
```

Comment:
Could we reuse ```mechanismName``` (https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/clients/admin/ScramMechanism.java#L79) so we don't need this duplicate replacement.

Reply:
Good suggestion, applied it.